### ISSUE TITLE
Adding example email to required email fields

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1909]
> The "Unsafe road conditions" and "Parking issue" email form fields automatically check for valid email addresses and report an error to the user if failing. The form labels need to include this information. Recommend changing email form labels to "Email* (example@example.com)".

## Comments

This can be done in the form field widget.

**Comment by jlyon** [comment]
Moving this to done, as this is the site-builder's responsibility to configure in the form builder @kevindherman 
![Uploading Screenshot - 2021-11-17T194949.700.png…]()
